interfaces/Portalicious/src/app/components/page-layout/components/breadcrumbs-title/breadcrumbs-title.component.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  input,
+} from '@angular/core';
+import { RouterModule } from '@angular/router';
+
+import { AppRoutes } from '~/app.routes';
+
+@Component({
+  selector: 'app-breadcrumbs-title',
+  standalone: true,
+  imports: [RouterModule],
+  templateUrl: './breadcrumbs-title.component.html',
+  styles: ``,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BreadcrumbsTitleComponent {
+  projectId = input.required<number>();
+  registrationId = input<number>();
+  registrationName = input<string>();
+  paymentDate = input<string>();
```

Comment:
I'm a bit confused by the choice of props here. Instead of passing in context-specific properties, and adding business logic to this component, I would rather pass in context-agnostic data, and move the logic to the declaration of the instances. Then you avoid all the `if`s in this file.

So, something like:

```suggestion
  parentLink = input<RouterLink['routerLink']>();
  parentTitle = input<string>();
  childTitle = input<string>();
```